src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java

```diff
@@ -240,6 +244,30 @@ protected void renderModules() {
             final String dir = RotationUtil.getDirection4D(false);
             renderText(dir, 2, height - 3 - RENDERER.getStringHeightI() * 2 - animationY);
         }
+
+        if (totems.getValue()) {
+            ScaledResolution sr = new ScaledResolution(Minecraft.getMinecraft());
```

Comment:
A cached ScaledResolution and its width and height are [here](https://github.com/3arthqu4ke/3arthh4ck/blob/1924ecd8e88067eff23856fb77d77eceebe7fe4e/src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java#L104) please use that one.